PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -163,11 +236,14 @@ void mapOemVirtualKeys()
 	// determine the active keyboard "language"
 	LANGID current_lang_id = LOWORD(GetKeyboardLayout(0));
 	
+	std::vector<UCHAR> localizableKeypad{VK_MULTIPLY, VK_ADD, VK_SUBTRACT, VK_DECIMAL, VK_DIVIDE, VK_SEPARATOR};

```

Comment:
By "keypad" do you mean this part?
https://en.wikipedia.org/wiki/Keypad#/media/File:Numpad.svg

Reply:
Sorry, yes, in my mind it's the "numeric keypad" or just "keypad"; I had forgotten you use the term "numpad".  I will update the name.